## ISMB 2019 Tutorial on "Recent Advances in Statistical Methods and Computational Algorithms for Single-Cell Omics Analysis"

This tutorial is focused on advanced statistical and computational methods that are recently developed for 
single-cell omics data. It is intended for an audience with genomics/computational background, who are
interested in cutting-edge developments of single-cell research, including both method development and application. 

All tutorial materials and extra links are provided here.

### Schedule

Official schedule for this tutorial can be found [here](https://www.iscb.org/ismbeccb2019-program/tutorials#am2).

### Course Instructors (equal contributors; alphabetical order)

Rhonda Bacher, University of Florida, rbacher@ufl.edu

Yuchao Jiang, University of North Carolina-Chapel Hill, yuchaoj@email.unc.edu

Jingshu Wang, University of Chicago, wangjingshususan@gmail.com

Please contact any of us regarding comments or questions.

### Tutorial Feedback

If you attend our tutorial at ISMB 2019, please provide feedback via this survery:
https://goo.gl/forms/0sR1kfVO6nj4X8bO2


### Slides

[Single-Cell ATAC-Seq and Multimodal Alignment](https://github.com/rhondabacher/ISMB2019_SingleCellTutorial/blob/master/slides/5_multimodal_alignment.pdf)

[Single-Cell Cancer Genomics](https://github.com/rhondabacher/ISMB2019_SingleCellTutorial/blob/master/slides/6_cancer_genomics.pdf)

### List of Methods

#### Single-cell epigenomics

SCRAT: Single-cell regulome data analysis ([paper](https://academic.oup.com/bioinformatics/article/33/18/2930/3823309), [software](https://github.com/zji90/SCRAT))

scABC: Unsupervised clustering and epigenetic classification of single cells ([paper](https://www.nature.com/articles/s41467-018-04629-3),[software](https://github.com/SUwonglab/scABC))

Destin: Toolkit for single-cell analysis of chromatin accessibility ([paper](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz141/5367832),[software](https://github.com/urrutiag/destin))

#### Single-cell multimodal alignment

Duren, Zhana, et al. "Modeling gene regulation from paired expression and chromatin accessibility data." Proceedings of the National Academy of Sciences 114.25 (2017): E4914-E4923.

Welch, Joshua D., Alexander J. Hartemink, and Jan F. Prins. "MATCHER: manifold alignment reveals correspondence between single cell transcriptome and epigenome dynamics." Genome biology 18.1 (2017): 138.

Stuart, Tim, et al. "Comprehensive Integration of Single-Cell Data." Cell (2019).

Butler, Andrew, et al. "Integrating single-cell transcriptomic data across different conditions, technologies, and species." Nature biotechnology 36.5 (2018): 411.

Zhou, Weiqiang, Zhicheng Ji, and Hongkai Ji. "Global prediction of chromatin accessibility using RNA-seq from small number of cells." bioRxiv (2016): 035816.

Haghverdi, Laleh, et al. "Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors." Nature biotechnology 36.5 (2018): 421.

#### Single-cell cancer genomics

Jiang, Yuchao, et al. "Assessing intratumor heterogeneity and tracking longitudinal and spatial clonal evolutionary history by next-generation sequencing." Proceedings of the National Academy of Sciences 113.37 (2016): E5528-E5537.

Urrutia, Eugene, et al. "Integrative pipeline for profiling DNA copy number and inferring tumor phylogeny." Bioinformatics34.12 (2018): 2126-2128.

Tirosh, Itay, et al. "Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma." Nature539.7628 (2016): 309.

Fan, Jean, et al. "Linking transcriptional and genetic tumor heterogeneity through allele analysis of single-cell RNA-seq data." Genome research 28.8 (2018): 1217-1227.

McCarthy, Davis James, et al. "Cardelino: Integrating whole exomes and single-cell transcriptomes to reveal phenotypic impact of somatic variants." bioRxiv (2018): 413047.

Wang, Rujin, Dan-Yu Lin, and Yuchao Jiang. "SCOPE: a normalization and copy number estimation method for single-cell DNA sequencing." bioRxiv (2019): 594267.


### Other Resources

Analysis of single cell RNA-seq data: https://hemberg-lab.github.io/scRNA.seq.course

Awesome single cell: https://github.com/seandavi/awesome-single-cell

simpleSingleCell: http://bioconductor.org/packages/simpleSingleCell

10X Genomics single-cell expression: https://support.10xgenomics.com/single-cell-gene-expression/datasets

Single Cell Portal: https://portals.broadinstitute.org/single_cell

Single Cell Expression Atlas: https://www.ebi.ac.uk/gxa/sc

conquer: http://imlspenticton.uzh.ch:3838/conquer

Human Cell Atlas: https://www.humancellatlas.org/

Mouse Cell Atlas: http://bis.zju.edu.cn/MCA 
